app.rb

```diff
@@ -1,15 +1,16 @@
 require 'sinatra/base'
-require_relative './lib/user'
 require_relative 'lib/database_connection_setup'
 
 class MakersBNB < Sinatra::Base
   enable :sessions
 
```

Comment:
Possible suggestion for the controller as a whole: grouping the routes according to feature with comment headings. Might improve readability.

Reply:
Great idea 💯